source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,120 @@
 Desktop Application Changelog
 ========================================
 
+Release 4.0.0
+--------------
+
+Release date: January 23, 2018
+
+This release contains multiple security updates for Windows, Mac and Linux, and it is highly recommended that users upgrade to this version.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added a dialog to allow the user to reopen the desktop app if it quits unexpectedly.
+- Mattermost animation icon is now displayed when loading a page, instead of a blank screen.
+- Added a dialog to request permissions to show desktop notifications or to use microphone and video for video calls from untrusted origins.
+- The "Saved" indicator now appears for both Server Management and App Options on the Settings page.
+- Close button on the Settings page now has a hover effect.
+- Added new admin configuration settings for:
+   - Disabling server management where the user cannot add or edit the server URL.
+   - Setting one or more pre-configured server URLs for the end user.
+   - Customizing the link in **Help > Learn More..**.
+
+Windows
+^^^^^^^^^^^^^
+
+- Added support for protocol deep linking where the desktop app opens via `mattermost://` link if app is already installed.
+- Added the ability to more easily white-label the Mattermost taskbar icon on custom builds.
+
+Mac
+^^^^^^^^^^^^^
+
+- Added support for protocol deep linking where the desktop app opens via `mattermost://` link if app is already installed.
+- Added `Ctrl+Tab` and `Ctrl+Shift+Tab` shortcuts to switch between server tabs.
+- Added the option to bounce the Dock icon when receiving a notification.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron from v1.6.11 to v1.7.9. Electron is the underlying technology used to build the Desktop apps.
```

Comment:
Ah, this was updated to v1.7.11 this week. Needs updating in the desktop app repo too.